File: despatch.Domain/Models/Default/Request/UserOrganizationsRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace despatch.Domain.Models.Default.Request
{
    public class UserOrganizationsRequest : BaseEntityRequest
    {
        public string ContactDataId { get; set; }
        public string OrganizationId { get; set; }

        public virtual UserRequest User { get; set; }
        public virtual OrganizationRequest Organization { get; set; }
    }
}

File: despatch.WEB/app/src/swagger/models/user-organizations-request.ts
/* tslint:disable */
import { BaseEntityRequest } from './base-entity-request';
import { OrganizationRequest } from './organization-request';
import { UserRequest } from './user-request';
export interface UserOrganizationsRequest extends BaseEntityRequest{
  contactDataId?: string;
  organization?: OrganizationRequest;
  organizationId?: string;
  user?: UserRequest;
}
